cmd/user/internal/domain/user/commands.go

```diff
@@ -57,9 +58,8 @@ func NewCommandFromPayload(contract string, payload []byte) (domain.Command, err
 
 // RequestAccessToken command
 type RequestAccessToken struct {
-	ID       uuid.UUID `json:"id"`
-	Email    string    `json:"email"`
-	Password string    `json:"password"`
+	Email    string `json:"email"`
+	Password string `json:"password"`
```

Comment:
The intention was here to allow user to login using "magic link" which seems to be very trendy way of doing things this days.

How does it work ?
User simply provides his email address and clicks login where he gets an email with link including auth token

this way password is not required and this was my intention when i wrote this solution here

Reply:
Yep it's a valid point, just did some research and it seems like the best way to go. It can also feature sms if needed in the future as well. Will revert back to the original passwordless solution.